rstar/src/primitives/mod.rs

```diff
@@ -1,9 +1,11 @@
 //! Contains primitives ready for insertion into an r-tree.
 
+mod geomwithdata;
```

Comment:
I think snake case `geom_with_data` as done for `point_with_data` would actually be more readable here.